lib/internal/test_runner/test.js

```diff
@@ -639,12 +647,18 @@ class Test extends AsyncResource {
         return;
       }
 
-      await afterEach();
-      await after();
+      // Call pass first so the result is accessible in the after hooks.
```

Comment:
nitpick, assuming there are tests I feel this comment is redundant

Reply:
Alright, I'll remove